### ISSUE TITLE
Delete files_videoviewer since files_videoplayer is not bundled?

### ISSUE THREAD

**Issue by PVince81** [owncloud/apps#2203]
Since recent OC versions, the video viewer has been replaced by https://github.com/owncloud/files_videoplayer

To avoid confusion I suggest that we remove "files_videoviewer" from this repository, or at least move it into an "archive" or "unmaintained" folder. Unless there's still a maintainer for it ?

@DeepDiver1975 @RealRancor

**Comment by RealRancor** [comment]
Suggestion:

1. Move all apps which are active / shipped with oC into own repositories like suggested in https://github.com/owncloud/apps/issues/2096

2. Move the /apps repository to https://github.com/owncloudarchive/

**Comment by PVince81** [comment]
Sounds good.

From what I saw there are apps which are still used like "user_saml" that get a PR or two a year, so might want to consider such as active too.

**Comment by RealRancor** [comment]
Yeah, needs some work or differ between unmaintained / broken apps and the ones still in use and working.
